# Typ2docx heading 1: body size and bold (pristine was 20 pt, not
# bold); left aligned is Word's default and stays. Heading1Char
# mirrors.
--- a/word/styles.xml
+++ b/word/styles.xml
@@ -208,8 +208,9 @@
       w:eastAsiaTheme="majorEastAsia" w:hAnsiTheme="majorHAnsi"
       w:cstheme="majorBidi" />
       <w:color w:val="000000" />
-      <w:sz w:val="40" />
-      <w:szCs w:val="40" />
+      <w:b />
+      <w:sz w:val="24" />
+      <w:szCs w:val="24" />
     </w:rPr>
   </w:style>
   <w:style w:type="paragraph" w:styleId="Heading2">
@@ -410,8 +411,9 @@
       w:eastAsiaTheme="majorEastAsia" w:hAnsiTheme="majorHAnsi"
       w:cstheme="majorBidi" />
       <w:color w:val="000000" />
-      <w:sz w:val="40" />
-      <w:szCs w:val="40" />
+      <w:b />
+      <w:sz w:val="24" />
+      <w:szCs w:val="24" />
     </w:rPr>
   </w:style>
   <w:style w:type="character" w:customStyle="1"
